src/components/ui/Chat/ChannelList/ChannelItem.tsx

```diff
@@ -22,25 +22,80 @@ export interface ChannelItemProps
   isSelected?: boolean;
   /** Callback function when clicked */
   onClick: () => void;
-  /** Signifies of there are unread message in the channel. */
+  /** Signifies if there are unread message in the channel. */
   unread?: boolean;
   /** Content of the badge signaling unread messages, such as a badge displaying the number. */
   unreadBadgeLabel?: string;
+  /** Indicate if we want to display the detailed channel view with last channel message or timestamp */
+  displayDetailedChannelView?: boolean;
+  /* Content of the last channel message to be displayed */
+  lastChannelMessage?: string;
```

Comment:
I was thinking of props more like:

previewString?: string,
timestamp?: number (or Date)

and then in the implementation below:
```
if timestamp -> render the timestamp and actions dropdown icon, otherwise, render just the action dropdown

if previewString -> render a div with the channel name and previewString , otherwise render only the channel button
```